src/coredump/coredumpctl.c

```diff
@@ -467,6 +495,21 @@ static void analyze_coredump_file(
         *ret_size = UINT64_MAX;
 }
 
+/*static int resolve_filename(const char *prefix, char **filename) {*/
+static int resolve_filename(const char *path, const char *root, char **ret) {
+        char *resolved = NULL;
+        int r;
+
+        if (!path)
+                return 0;
+
+        r = chase_symlinks(path, root, CHASE_PREFIX_ROOT, &resolved, NULL);
```

Comment:
The coredump file might not exist anymore, so we should add `CHASE_NONEXISTENT` here so we don't fail if the file doesn't exist.

If `CHASE_NONEXISTENT` is set, `chase_symlinks()` will return 0 if the file doesn't exist and 1 if it does. So you should return this returncode from `resolve_filename()` and then for all callers, depending on whether they expect the file to exist or not, handle the error appropriately.